erpnext/patches/v12_0/update_is_cancelled_field.py

```diff
@@ -3,14 +3,21 @@
 
 
 def execute():
-	try:
-		frappe.db.sql("UPDATE `tabStock Ledger Entry` SET is_cancelled = 0 where is_cancelled in ('', NULL, 'No')")
-		frappe.db.sql("UPDATE `tabSerial No` SET is_cancelled = 0 where is_cancelled in ('', NULL, 'No')")
+	#handle type casting for is_cancelled field
 
-		frappe.db.sql("UPDATE `tabStock Ledger Entry` SET is_cancelled = 1 where is_cancelled = 'Yes'")
-		frappe.db.sql("UPDATE `tabSerial No` SET is_cancelled = 1 where is_cancelled = 'Yes'")
+	for doc_mapper in (('stock','Stock Ledger Entry'),
+		('stock','Serial No'),
+		('accounts','GL Entry')):
```

Comment:
@deepeshgarg007 @saurabh6790 

GL entry has never had this select field for `is_cancelled` in any production version prior to v13; we need to check if this column even exists before running the query or patch will fail. 

Is this to accommodate any beta branch which had yes/no for is_cancelled?

Reply:
Not remembering  the site but yes added this as it was breaking in v11 to v13 migrations.